common/utilities/setQueryParameters.js

```diff
@@ -0,0 +1,13 @@
+import Router from 'next/router'
+
+export default queries => {
+  if(!process.browser){
```

Comment:
this should be in a neat function in another file `isBrowser()` `isServer()` I've done this in other code bases (maybe function-camp)